src/OpenSage.Game/OpenSage.Game.csproj

```diff
@@ -21,7 +21,7 @@
     <PackageReference Include="Veldrid.RenderDoc" Version="1.0.0" />
     <PackageReference Include="Veldrid.SDL2" Version="$(VeldridVersion)" />
     <PackageReference Include="Veldrid.StartupUtilities" Version="$(VeldridVersion)" />
-    <PackageReference Include="Veldrid.SPIRV" Version="1.0.12" />
+    <PackageReference Include="Veldrid.SPIRV" Version="1.0.11" />
```

Comment:
According to Eric he's added a backwards incompatible change in 1.0.12 that he'll rever in 1.0.13